children/models.py

```diff
@@ -83,6 +83,11 @@ class Child(UUIDPrimaryKeyModel, TimestampedModel, GDPRModel, SerializableMixin)
         related_name="children",
         blank=True,
     )
+    notes = models.TextField(
+        verbose_name=_("notes"),
+        blank=True,
+        help_text=_("Notes related to child, e.g. child's attendance history"),
+    )
```

Comment:
I think the name "notes" is too generic for this feature, because how I've understood this, this new field is only for "history of event participations". How about "attendance_history" or "attendance_history_notes" or "event_history_notes"?



```suggestion
    attendance_history = models.TextField(
        verbose_name=_("notes"),
        blank=True,
        help_text=_("Notes related to child, e.g. child's attendance history"),
    )
```


I think this name "notes" is good if we decide to have a note field that would be available in the child edit form (in the UI), but I think this should be a whole new query and mutation, only for this one purpose.